### ISSUE TITLE
can't get programmatic access working

### ISSUE THREAD

**Issue by raistlinthewiz** [balmasi/migrate-mongoose#4]
tried programming access as you explained in #2 

```
const migration = require('migrate-mongoose');
		var migrator = new migration({
			dbConnectionUri: util.format('mongodb://%s/%s', config.get('database.mongodb.host'), config.get('database.mongodb.name'))
		});
```

it seems the package.json sets the default file as cli.js and cli is firing;

```
Not enough non-option arguments: got 0, need at least 1
```

how can instead access lib.js instead?

**Comment by balmasi** [comment]
Hey I updated package json.

The reason I didn't notice this is because in my scripts I usually just use seomthing like this

`./node_modules/.bin/migrate --autosync --es6 -d mongodb://localhost/dev` 

cause that migrate bin is executable.

Upgrading to v2.0 fixes the package.json. #

**Comment by bonesoul** [comment]
updated the package and now getting this;

```
  - module.js:455 Function.Module._resolveFilename
    module.js:455:15

  - module.js:403 Function.Module._load
    module.js:403:25

  - module.js:483 Module.require
    module.js:483:17

  - module.js:20 require
    internal/module.js:20:19

  - lib.js:7 Object.<anonymous>
    [buharmetre]/[migrate-mongoose]/dist/lib.js:7:21

  - module.js:556 Module._compile
    module.js:556:32

  - module.js:565 Object.Module._extensions..js
    module.js:565:10

  - module.js:473 Module.load
    module.js:473:32

  - module.js:432 tryModuleLoad
    module.js:432:12

  - module.js:424 Function.Module._load
    module.js:424:3

  - module.js:483 Module.require
    module.js:483:17

  - module.js:20 require
    internal/module.js:20:19

  - migration.js:10 Object.<anonymous>
    E:/Coding/buharmetre/src/lib/migration/migration.js:10:19

  - module.js:556 Module._compile
    module.js:556:32

  - module.js:565 Object.Module._extensions..js
    module.js:565:10

  - module.js:473 Module.load
    module.js:473:32

  - module.js:432 tryModuleLoad
    module.js:432:12

  - module.js:424 Function.Module._load
    module.js:424:3

  - module.js:483 Module.require
    module.js:483:17

  - module.js:20 require
    internal/module.js:20:19

  - app.js:62
    E:/Coding/buharmetre/src/app.js:62:4

  - debuggability.js:299 Promise._execute
    [buharmetre]/[bluebird]/js/release/debuggability.js:299:9
```

**Comment by balmasi** [comment]
@bonesoul good call. Was exporting under property 'default' (ES6)
Fixed in 2.02 update (available now)

Thanks again!
